cli/flox/src/commands/environment.rs

```diff
@@ -1244,18 +1256,115 @@ impl Update {
     pub async fn handle(self, flox: Flox) -> Result<()> {
         subcommand_metric!("update");
 
-        let message = if self.global {
-            self.update_global_manifest(flox)?
-        } else {
-            self.update_manifest(flox)?
+        let mut description = None;
+        let (old_lockfile, new_lockfile) = match self.environment_or_global {
+            EnvironmentOrGlobalSelect::Environment(ref environment_select) => {
+                let concrete_environment =
+                    environment_select.detect_concrete_environment(&flox, "update")?;
+
+                description = Some(environment_description(&concrete_environment)?);
+
+                self.update_manifest(flox, concrete_environment)?
+            },
+            EnvironmentOrGlobalSelect::Global => self.update_global_manifest(flox)?,
         };
 
-        info!("{}", message);
+        let global = matches!(
+            self.environment_or_global,
+            EnvironmentOrGlobalSelect::Global
+        );
+
+        if let Some(ref old_lockfile) = old_lockfile {
+            if new_lockfile.registry.inputs == old_lockfile.registry.inputs {
```

Comment:
suggestion: make `description` and `global` in the same match block
At least description as an optinal is a bit weird since you will always unwrap it when using it.
Maybe we can find a sentene structure that allows reusing the description for global manifests as well...


```suggestion
        subcommand_metric!("update");

        let ((old_lockfile, new_lockfile), description, global) = match self.environment_or_global {
            EnvironmentOrGlobalSelect::Environment(ref environment_select) => {
                let concrete_environment =
                    environment_select.detect_concrete_environment(&flox, "update")?;
                let description = environment_description(&concrete_environment)?;
                (
                    self.update_manifest(flox, concrete_environment)?,
                    description,
                    false,
                )
            },
            EnvironmentOrGlobalSelect::Global => (
                self.update_global_manifest(flox)?,
                "global manifest".to_string(),
                true,
            ),
        };

        if let Some(ref old_lockfile) = old_lockfile {
```

Reply:
Moving `description` and `global` into the match block. Open to suggestions to get rid of the unwrap as I don't think it's the nicest, but I want to prioritize user facing sentence structure rather than removing unwraps